src/note.jl

```diff
@@ -101,6 +101,7 @@ Base.firstindex(n::Notes) = firstindex(n.notes)
 Base.getindex(n::Notes, i::Int) = n.notes[i]
 Base.getindex(n::Notes, r) = Notes(n.notes[r], n.tpq)
 Base.view(n::Notes, r) = view(n.notes, r)
+Base.keys(n::Notes) = 1:length(n)
```

Comment:
```suggestion
Base.eachindex(n::Notes) = eachindex(n.notes)
Base.keys(n::Notes) = eachindex(n)
```
Please use this version instead. It is better!

Reply:
I'll change this